migrations/036-add_customers_table.js

```diff
@@ -0,0 +1,32 @@
+var db = require('./db')
+
+exports.up = function(next) {
+  const sql =
+    [
+    'CREATE TABLE customers ( ' +
```

Comment:
Also, our practice is to not capitalize anything, all lowercase. But that's not a big deal.